Explorer/Assets/DCL/ResourcesUnloading/CacheCleaner.cs

```diff
@@ -65,15 +65,33 @@ public void UnloadCache()
             wearableStorage!.Unload(fpsCapBudget);
             emoteCache!.Unload(fpsCapBudget);
             gltfContainerAssetsCache!.Unload(fpsCapBudget, GLTF_UNLOAD_CHUNK);
+            lodCache!.Unload(fpsCapBudget, GLTF_UNLOAD_CHUNK);
             assetBundleCache!.Unload(fpsCapBudget, AB_UNLOAD_CHUNK);
             profileCache!.Unload(fpsCapBudget, PROFILE_UNLOAD_CHUNK);
             profileIntentionCache!.Unload(fpsCapBudget, PROFILE_UNLOAD_CHUNK);
-            lodCache!.Unload(fpsCapBudget, GLTF_UNLOAD_CHUNK);
             roadCache!.Unload(fpsCapBudget, GLTF_UNLOAD_CHUNK);
 
             ClearExtendedObjectPools();
         }
 
+        public void UnloadCacheImmediate()
```

Comment:
Maintaining two methods like these with the same ordering is also error-prone. It's another argument again the second way